helm-chart/renku/templates/authz-secrets.yaml

```diff
@@ -0,0 +1,32 @@
+---
+{{- $db_password := randAlphaNum 64 }}
+{{- $db_password_enc := $db_password | b64enc | quote }}
+{{- $db_conn_uri := (printf "postgres://authz:%s@%s:5432/authz" $db_password (include "postgresql.fullname" .)) | b64enc | quote }}
+{{- $api_key := randAlphaNum 64 | b64enc | quote }}
+
+{{- $renkuFullname := include "renku.fullname" . -}}
+
+{{- $secretName := cat $renkuFullname "-authz" | nospace }}
+{{- $secret := (lookup "v1" "Secret" .Release.Namespace $secretName) }}
+{{- if $secret }}
+{{- $db_password_enc = index $secret.data "SPICEDB_POSTGRES_PASSWORD" }}
+{{- $db_conn_uri = index $secret.data "SPICEDB_DATASTORE_CONN_URI" }}
+{{- $api_key = index $secret.data "SPICEDB_GRPC_PRESHARED_KEY" }}
+{{- end -}}
```

Comment:
This doesn't seem to allow for existing postgres credentials to be provided in the values file

Reply:
This is similar to the database name. I do not want to allow people to change this. What we accept is a single postgres admin password that can be passed through the values file. Then using this password renku makes different postgres accounts with their own passwords and gives them access to specific hardcoded databases. If I accept this in the values file I also have to add logic and jobs to compare it with a previous value and to then if needed update the database so that everything works.